readme.md

```diff
@@ -67,7 +67,7 @@ To use each plugin-specific configuration, you'll need to do the following:
 
 ### Plugin-specific configurations
 
-**All plugin-specific configurations extend the same base configuration, found in `base.js`.**
+**All plugin-specific configurations extend the same base configuration (except for `xss`), found in `base.js`.**
```

Comment:
No one will know quite what it means to extend base.js. I'd advocate removing this line from the readme, and simply stating in the description of xss.js that it does not perform any JS style linting if used on its own.